docs/guidance/map/connecting_publications.md

```diff
@@ -0,0 +1,429 @@
+
+The publishers would need to agree the process and responsibilities for minting ocids. For example: who can mint ocids; if an ocid cannot be calculated deterministically from the input data, then how to lookup whether an ocid for a given process already exists, and how to share the new ocid for others to find. They would also need to implement the agreed approach, whether this means reporting a new ocid to a central registry, or requesting the ocid for a given procedure from a shared "ocid issuer" service, etc.
+
+Building on the PCS and FTS example above, if PCS were to publish only the `parties` array and `tender` object and FTS were to publish only the `awards` array, then it would be possible for FTS to reuse the ocids minted by PCS:
+
+`````{tab-set}
+````{tab-item}  PCS release
+```json
+{
+  "id": "rls-1-JAN468109",
+  "ocid": "ocds-r6ebe6-0000710562",
+  "date": "2023-01-09T00:00:00Z",
+  "tag": [
+    "tender"
+  ],
+  "parties": [
+    {
+      "id": "org-1",
+      "identifier": {
+        "legalName": "Fife Council"
+      },
+      "name": "Fife Council",
+      "roles": [
+        "buyer"
+      ]
+    }
+  ], 
+  "tender": {
+    "title": "New build housing and land acquisition (Fife Wide)",
+    "mainProcurementCategory": "works",
+    "value": {
+      "amount": 100000000,
+      "currency": "GBP"
+    }
+  }
+}
+```
+````
+````{tab-item}  FTS release
+```json
+{
+  "id": "000521-2023",
+  "ocid": "ocds-r6ebe6-0000710562",
+  "date": "2023-01-09T12:26:29Z",
+  "tag": [
+    "award"
+  ],  
+  "awards": [
+    {
+      "id": "1",
+      "title": "New build housing and land acquisition (Fife Wide)",
+      "value": {
+        "amount": 75000000,
+        "currency": "GBP"
+      }
+    }
+  ]
+}
+```
+````
+````{tab-item}  Compiled release
+```json
+{
+  "ocid": "ocds-r6ebe6-0000710562",
+  "parties": [
+    {
+      "id": "org-1",
+      "identifier": {
+        "legalName": "Fife Council"
+      },
+      "name": "Fife Council",
+      "roles": [
+        "buyer"
+      ]
+    }
+  ],
+  "tender": {
+    "title": "New build housing and land acquisition (Fife Wide)",
+    "mainProcurementCategory": "works",
+    "value": {
+      "amount": 100000000,
+      "currency": "GBP"
+    }
+  },
+  "awards": [
+    {
+      "id": "1",
+      "title": "New build housing and land acquisition (Fife Wide)",
+      "value": {
+        "amount": 75000000,
+        "currency": "GBP"
+      }
+    }
+  ]
+}
+```
+````
+`````
+
+However, this scenario is unlikely to occur since a publication that includes the `awards` array ought to also include details of the suppliers in the `parties` array. In which case, in order to avoid the risk of clashing identifiers, the publications would need to coordinate their approaches to assigning local identifiers to objects in the `parties` array.
```

Comment:
This paragraph is kind of saying  "gotcha!". Should we either:

- State upfront that they'll need to coordinate party IDs, before the example
- Use an example that requires no coordination (e.g. planning and tender)

Reply:
Haha, it is a bit like that!

If they need to coordinate party IDs, then it'll become like the next example (publications with consistent mappings of shared fields) so I don't think we should do that.

I was struggling to come up with a realistic example for publications with disjoint coverage, but I had misremembered that `planning` and `tender` could no longer co-exist in the same process. On re-reading the [contracting processes and planning processes example](https://standard.open-contracting.org/staging/1430-linking-releases-guidance/en/guidance/map/contracting_planning_processes/), I see that `planning` and `tender` can co-exist in the same planning process so I think we have two options for the 'publications with distinct coverage' example:

1. Update it to be about a planning process with two publishers: FTS publishes details of planned tenders in `tender` and HM Treasury publishes budget data in `planning.budget`. I'm still not sure this is a very realistic example
2. Replace it with a sentence like "However, this scenario is unlikely to occur since most publications are likely to have some fields in common, even when publishing data about different stages of the contracting (or planning) process."

Reply:
Hmm, perhaps the example given in https://github.com/open-contracting/standard/issues/364#issuecomment-1182420032 would be a better one to use, though we might need to adapt it to use the same context as the other examples on the page.

Reply:
@yolile Since you shared the example in the cited comment, do you have an opinion on what example is best here? 

We're looking for an example where two publications cover distinct fields, such that they can reuse OCIDs without e.g. coordination around IDs values in arrays.

If we can't think of a good example, I think we can put the "However," caveat in a note admonition at the start of the example, and encourage people to read the next example.

Reply:
From my example, there are some cases where the Chile Compra publication indeed doesn't have the contract data, because the buyers fill out that information using the Ministry of Works systems and not Chile Compra. There are other cases where contract info exists in both and they need to keep IDs in sync. But I guess we could use the first case for the example purposes. They are not publishing this yet (they should by the end of this month) but I think I  could find some test data if it is useful

Reply:
@yolile Assuming that Chile Compra has the award data, I would expect that there would still be some overlapping fields in the first case, e.g. `Contracts.awardID` and supplier data in `parties`, so it would be good to see some test data and to understand any overlapping fields are coordinated.

Reply:
@yolile just checking that you saw my previous comment - if you can share test data, it would be good to see it in order to understand a little more about the Chile example. If not, we can go with the approach suggested by James.

Reply:
Ah sorry, @duncandewhurst this one was lost in the multiple updates I'm getting from this and OC4IDS repos :) 

Now we have examples from a test server, for example: https://obrapublica.azurewebsites.net/Contratos/StandarOCDS?Id=346979 (you can check the publisher field in each release to check whether the releases are generated by Chile Compra ("Dirección de Compras y Contratación Pública") or by the Ministry of Public Works ("Observatorio del Gasto Fiscal - Dirección General de Obras Públicas"))

Let me know if this helps.

Reply:
No worries :-) I know that feeling!

The example is very helpful, but as I expected there are overlapping fields and it demonstrates coordination since the values of the fields are the same. For example, the award release from Chile Compra and the contract release from the Ministry of Public Works list the same organizations in the `parties` array ([notebook](https://colab.research.google.com/drive/10Woxkm4vN2p23dSCrj0xaqjxcPHiPdn3#scrollTo=E9wK0k6QAfLd)).

As such, I'll go with James's suggestion from https://github.com/open-contracting/standard/pull/1646#discussion_r1381176507

Reply:
Done in https://github.com/open-contracting/standard/pull/1646/commits/413c39012e15236ef8bb347b3b41df947e6e4c71